### DISCUSSION TITLE
图片预览还是不太方便

### DISCUSSION THREAD

**Discussion by J1nVel** [QL-Win/QuickLook#1157] [Ideas]
快速预览图片功能很好用，尤其是放大缩小功能很方便。但是图片放大或者之后，想看下一张图的话就必须恢复图片原有大小，甚至很多时候恢复图片原来尺寸之后也不能看下一张图。想要看下一张图，就得关闭当前的预览窗口，点击下一张图，然后敲击空格键才能看到。希望这个放大缩小功能不影响用键盘上下左右键查看上一张或者下一张图的功能。

**Comment by xupefei** [comment]
不用关闭预览，在文件管理器里选另一个文件就行了。

**Reply by X1asky** [reply]
确实，即使在焦点窗口，↑↓方向键也不管用，得用鼠标点其他的。

**Comment by tylvn** [comment]
v3.7.2 可以在不恢复图片原有大小的情况下，用↑↓←→方向键来切换照片。
